src/sysfiles.c

```diff
@@ -440,6 +447,119 @@ Int SyLoadModule( const Char * name, InitInfoFunc * func )
 
 #endif
 
+/****************************************************************************
+**
+*F * * * * * * * * * * finding location of executable * * * * * * * * * * * *
+*/
+
+/****************************************************************************
+** The function 'find_yourself' is based on code (C) 2015 Mark Whitis, under
+** the MIT License : http://stackoverflow.com/a/34271901/928031
+*/
+
+void find_yourself(const char * argv0, char * result, size_t size_of_result)
+{
+    char findyourself_save_path[LONG_PATH_SIZE] = { 0 };
+
+
+    strlcpy(findyourself_save_path, getenv("PATH"),
+            sizeof(findyourself_save_path));
+
+    // + 256 gives us (lots) of slack to put the program name on the end
+    char newpath[LONG_PATH_SIZE + 256];
+    char newpath2[LONG_PATH_SIZE + 256];
```

Comment:
Those variable names are not very helpful. Looking at the code, it seems that generally `newpath2` is used a a auxiliary buffer to construct new paths, so perhaps call it `aux_path` or `tmp_path` or so?

And `newpath` is used to store the result of `realpath`, so perhaps `resolved_path`?